Source/WebCore/editing/mac/EditorMac.mm

```diff
@@ -54,14 +54,14 @@
 #import "PlatformStrategies.h"
 #import "RenderBlock.h"
 #import "RenderImage.h"
-#import "RuntimeApplicationChecks.h"
 #import "SharedBuffer.h"
 #import "WebContentReader.h"
 #import "WebNSAttributedStringExtras.h"
 #import "markup.h"
 #import <AppKit/AppKit.h>
 #import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
 #import <wtf/RetainPtr.h>
+#import <wtf/RuntimeApplicationChecks.h>
```

Comment:
Sort of seems like we would want the Cocoa version, but I guess not!

Reply:
Yeah, the non-Cocoa one is sufficient.